tests/travis/composer.orm2.3.json

```diff
@@ -14,7 +14,8 @@
         "doctrine/orm": "<2.4"
     },
     "require-dev": {
-        "phpunit/phpunit": ">=4.8",
+        "phpunit/phpunit": "<5.0",
```

Comment:
PHPUnit 5.0+ is only supported on PHP 5.6+. With the require ">=4.8" composer was installing PHPUnit 5.0 when running the tests under PHP 5.6. This generates warnings like https://travis-ci.org/creof/doctrine2-spatial/jobs/89088715. Pushing PHP requirement to 5.6+ would solve a number of problems but also upset people I imagine.